src/test/java/stepDefinitions/disclosureurlSteps.java

```diff
@@ -0,0 +1,45 @@
+package stepDefinitions;
+
+import static org.junit.Assert.assertTrue;
+import static org.testng.Assert.assertTrue;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import pageObjects.listingDetails_page;
+import pageObjects.search_page;
+
+public class disclosureurlSteps {
+	public WebDriver driver;
+	private WebElement element;
+	public disclosureurlSteps(){
+		driver = hooks.driver;
+	}
+	
+	@Given("^I am on listing details page of listing with CHPL ID \"(.*)\"$")
+	public void i_am_on_listing_details_page_of_listing_with_CHPL_ID(String CHPL_ID) throws Throwable {
+		driver.get("https://chpl.ahrqstg.org/#/search");
```

Comment:
Why doesn't this just go straight to the details page? This step depends on the Search page, which means it's fragile